docs/generate_py_docs.py

```diff
@@ -0,0 +1,174 @@
+import re
+import os
+from textwrap import dedent, indent
+from os import listdir
+from os.path import isfile, join
+from bs4 import BeautifulSoup
+from docutils import core
+
+
+def rst2html(input_string: str, source_path: str = None, destination_path: str = None,
+             input_encoding: str = 'unicode', doctitle: bool = True,
+             initial_header_level: int = 1):
+    overrides = {'input_encoding': input_encoding,
+                 'doctitle_xform': doctitle,
+                 'initial_header_level': initial_header_level}
+    parts = core.publish_parts(
+        source=input_string, source_path=source_path,
+        destination_path=destination_path,
+        writer_name='html', settings_overrides=overrides)
+    return parts['body']
+
+
+def html2txt(html: str):
+    html_elements = BeautifulSoup(html, features="html.parser")
+
+    for tag in html_elements.select('style'):
+        tag.decompose()
+
+    for tag in html_elements.select('td'):
+        # remove non-breaking spaces
+        if tag.text == u'\xa0':
+            tag.decompose()
+        else:
+            # add new line and indent for the cells inside a table
+            if 'class' not in tag.attrs:
```

Comment:
Hmm is having `class` attribute equivalent to being in a table in Sphinx-generated HTML?

Reply:
No, it's just that these <td> tags without the 'class' attribute need indent